implementations/toUpperCase.js

```diff
@@ -0,0 +1,19 @@
+/*
+  The toLowerCase() method returns the value of the string converted to lower case.
+  toLowerCase() does not affect the value of the string str itself.
+
+  MDN Link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+
+  Characters from A-Z have ASCII code from 65 - 90.
+  And characters from a-z have ASCII code from 97-122.
+  We're checking this condition to implement this function.
+*/
```

Comment:
I think you are not focusing properly, the documentation seems messed up. Please do not do such mistakes. Also, you have to remove `split()` because it can have its own implementation.

Reply:
@ashokdey Sorry, it will not happen again. And I've removed split function as well.